claim/schema.py

```diff
@@ -112,16 +117,26 @@ def resolve_claim_attachments(self, info, **kwargs):
         if not info.context.user.has_perms(ClaimConfig.gql_query_claims_perms):
             raise PermissionDenied(_("unauthorized"))
 
-    def resolve_claim_admins(self, info, search=None, **kwargs):
+    def resolve_claim_admins(
+            self,
+            info,
+            search=None,
+            user_health_facility=None,
+            **kwargs
+    ):
         if not info.context.user.has_perms(ClaimConfig.gql_query_claim_admins_perms):
             raise PermissionDenied(_("unauthorized"))
 
+        queryset = ClaimAdmin.objects
```

Comment:
Please change to 
```suggestion
        queryset = ClaimAdmin.objects.all()
```

Reply:
okay, changed

Reply:
Quick explanation for logic behind adding `.all()` is that `ClaimAdmin.objects` is a [Manager](https://docs.djangoproject.com/en/4.0/topics/db/managers/) and .all() calls for QuerySet object. Let's not update every other query right now (as in some cases this might be intentional), but when we come across it in the further development please introduce this change.